### PR TITLE
Changing ENOS action plugin license from BSD to GPL

### PR DESCRIPTION
**Pull request by amuraleedhar**
##### SUMMARY
Changing BSD to GPL for license as Suggested by John

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME
lib/ansible/plugins/action/enos.py

##### ANSIBLE VERSION
ansible 2.5.0
config file = /etc/ansible/ansible.cfg
configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible-2.5.0-py2.7.egg/ansible
executable location = /usr/local/bin/ansible
python version = 2.7.6 (default, Oct 26 2016, 20:30:19) [GCC 4.8.4]

##### ADDITIONAL INFORMATION
This is an effort to support Lenovo ENOS Switches in Ansible

### CODE REVIEW

File: lib/ansible/plugins/action/enos.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # This code is part of Ansible, but is an independent component.
-# This particular file snippet, and this file snippet only, is BSD licensed.
+# This particular file snippet, and this file snippet only, is GPL licensed.
```
Comment (sivel) [inline lib/ansible/plugins/action/enos.py]: This change is not valid.  It is only updating a single word within the license, and not updating the full license header.

Additionally, in order to do this, you will need to get approval from the following additional authors to relicense the code:

Qalthos
ganeshrn

**Review by sivel — CHANGES_REQUESTED** [review]

File: lib/ansible/plugins/action/enos.py (hunk 1 above)
Comment (amuraleedhar) [reply lib/ansible/plugins/action/enos.py]: Ok. Then I will do it tomorrow morning my time. Thanks for this.

**Review by Qalthos — APPROVED** [review]
:+1: to relicense

**Review by bcoca — CHANGES_REQUESTED** [review]
FYI, we normally require files in module_utils/ to be BSD licensed, the change normally goes in the opposite direction

**Comment by amuraleedhar** [comment]
If BSD License is OK and GPL is not required, we can abandon this PR as the file is already bearing a BSD license

**Comment by ganeshrn** [comment]
@amuraleedhar @bcoca As the change is in `plugins/action/` the change from BSD to GPL license is required

**Review by ganeshrn — APPROVED** [review]

**Comment by amuraleedhar** [comment]
Please merge this

Thanks & Regards
-Anil

**Comment by ganeshrn** [comment]
@amuraleedhar Approved offline by @bcoca. Merging it now.
Thank you.

**Pull request merged**